propka/atom.py

```diff
@@ -5,15 +5,36 @@
 from . import hybrid36
 
 
+# Format strings that get used in multiple places (or are very complex)
+PKA_FMT = "{:6.2f}"
+INPUT_LINE_FMT = (
+    "{type:6s}{r.numb:>5d} {atom_label} {r.res_name}{r.chain_id:>2s}"
+    "{r.res_num:>4d}{r.x:>12.3f}{r.y:>8.3f}{r.z:>8.3f}{group:>6s}{pka:>6s} \n")
+PDB_LINE_FMT1 = (
+    "{type:6s}{r.numb:>5d} {atom_label} {r.res_name}{r.chain_id:>2s}"
+    "{r.res_num:>4d}{r.x:>12.3f}{r.y:>8.3f}{r.z:>8.3f}{r.occ:>6s}"
+    "{r.beta:>6s}\n")
+MOL2_LINE_FMT = (
+    "{id:<4d} {atom_label:4s} "
+    "{r.x:>10.4f} {r.y:>10.4f} {r.z:>10.4f} "
+    "{r.sybyl_type:>6s} {r.res_num:>6d} {r.res_name:>10s}     0.0000\n")
+PDB_LINE_FMT2 = (
+    "ATOM {numb:>6d} {atom_label} {res_name}{chain_id:>2s}{res_num:>4d}"
+    "{x:>12.3f}{y:>8.3f}{z:>8.3f}{occ:>6.2f}{beta:>6.2f}\n")
+STR_FMT = (
+    "{r.numb:>5d}-{r.name:>4s} {r.res_num:>5d}-{r.res_name:>3s} "
+    "({r.chain_id:1s}) [{r.x:>8.3f} {r.y:>8.3f} {r.z:>8.3f}] {r.element:s}")
+
+
 class Atom(object):
     """Atom class - contains all atom information found in the PDB file"""
 
-    def __init__(self, line=None, verbose=False):
+    def __init__(self, line=None, _=False):
         """Initialize Atom object.
 
         Args:
             line:  Line from a PDB file to set properties of atom.
-            verbose:  TODO - this does not appear to be used.  Can we remove it?
+            _:  TODO - this does not appear to be used. Can we remove it?
```

Comment:
What are the plans for this TODO? Removing `verbose`/`_` sounds fine to me.

Reply:
Fixed in https://github.com/Electrostatics/propka-3.1/commit/a716c89a3641a7bb8d07b445c1fbb6e03685c93c